packages/next/src/server/config-shared.ts

```diff
@@ -202,6 +202,11 @@ export interface ReactCompilerOptions {
 export interface LoggingConfig {
   fetches?: {
     fullUrl?: boolean
+    /**
+     * If true, fetch requests that are restored from the HMR cache are logged
+     * during an HMR refresh request, i.e. when editing a server component.
+     */
+    hmrRefreshes?: boolean
```

Comment:
I think we need to update this in the `config-schema` as well here https://github.com/vercel/next.js/blob/242dc7e3ac248bcb6928f4383045c6388ffdfcf9/packages/next/src/server/config-schema.ts#L541

Reply:
Oh, of course. I wonder why we use `z.strictObject` only for some of the nested configs. 🤔 Otherwise `next dev` would have complained about this.